### PR TITLE
Copy notebooks in image

### PR DESCRIPTION
**Pull request by HazalCiplak**
part of https://github.com/elifesciences/data-hub-issues/issues/965

### CODE REVIEW

File: Dockerfile.airflow [hunk 1]
```diff
@@ -39,6 +39,7 @@ RUN mkdir $AIRFLOW_HOME/applications_file_directory
 ENV AIRFLOW_APPLICATIONS_DIRECTORY_PATH $AIRFLOW_HOME/applications_file_directory
 
 COPY data_science_pipeline ./data_science_pipeline
+COPY notebooks ./notebooks
```
Comment (de-code) [inline Dockerfile.airflow:42]: Might be worth deleting the line for the example notebook?

File: .env [hunk 2]
```diff
@@ -1,4 +1,5 @@
 COMPOSE_PROJECT_NAME=data-science-dags
+IMAGE_REPO=elifesciences/data-science-dags
```
Comment (de-code) [inline .env:2]: Didn't we already do that when pushing the image? Maybe something needs to be merged from develop or so?

Comment (HazalCiplak) [reply .env]: Ah yes!

**Comment by HazalCiplak** [comment]
Ok there was a bit confusion so I rolled back the changes. :)

**Pull request merged**